libraries/AP_Compass/AP_Compass_AK8963.cpp

```diff
@@ -226,6 +232,11 @@ bool AP_Compass_AK8963_MPU9250::init()
     if (_backend == NULL) {
         hal.scheduler->panic(PSTR("_backend coudln't be allocated"));
     }
+    if (!_backend->init()) {
```

Comment:
Not really. Inside the if it's a call to panic() so nothing else will be executed